jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/categoryattr/category-attributes-detail-view.xml

```diff
@@ -0,0 +1,241 @@
+                    <div width="100%" classNames="flex items-start">
+                        <codeEditor id="whereClauseField"
+                                    label="msg://io.jmix.dynattr.model/CategoryAttribute.whereClause"
+                                    dataContainer="categoryAttributeDc" property="whereClause"
+                                    width="100%"
+                                    showLineNumbers="true"
+                                    highlightGutterLine="false"
+                                    highlightActiveLine="false"
+                                    colspan="2"
+                                    mode="SQL"
+                                    showGutter="true"
+                                    height="12em"/>
+                    </div>
+                    <div width="100%" classNames="flex items-start">
+                        <codeEditor id="recalculationScriptField"
+                                    label="msg://io.jmix.dynattr.model/CategoryAttributeConfiguration.recalculationScript"
+                                    dataContainer="categoryAttributeDc" property="configuration.recalculationScript"
+                                    showLineNumbers="true"
+                                    width="100%"
+                                    highlightGutterLine="false"
+                                    highlightActiveLine="false"
+                                    colspan="2"
+                                    showGutter="true"
+                                    mode="GROOVY"
+                                    height="12em"/>
+                    </div>
+                </vbox>
+            </tab>
+            <tab id="localizationTab" label="msg://localizationTab.title"
+                 visible="false">
+                <vbox id="localizationContainer" padding="false" margin="false">
+                    <formLayout id="categoryAttrLocalizationForm" dataContainer="categoryAttributeDc" width="100%">
+                        <textField id="localizedNameField" property="name" readOnly="true"/>
+                        <textField id="localizedDescriptionField" property="description" readOnly="true"/>
+                    </formLayout>
+                </vbox>
+            </tab>
+            <tab id="visibilityTab" label="msg://visibilityTab.title">
+                <vbox padding="false" margin="false">
+                    <h3 text="msg://targetViews"/>
+                    <html content="msg://visibilityTab.help"/>
+                    <hbox classNames="buttons-panel">
+                        <button id="createBtn" action="targetScreensTable.create"/>
+                        <button id="removeBtn" action="targetScreensTable.remove"/>
+                    </hbox>
+                    <dataGrid id="targetScreensTable" dataContainer="targetScreensDc" width="100%" height="250px"
+                              editorBuffered="true" minWidth="55em">
```

Comment:
minWidth? Why not set `minWidth` for the entire view? This view isn't responsive any way
<img width="555" alt="Screenshot 2023-10-19 at 16 45 45" src="https://github.com/jmix-framework/jmix/assets/6077149/1a36c3ea-8563-48b3-b316-4a1e2cb63a3a">
<img width="392" alt="Screenshot 2023-10-19 at 16 46 53" src="https://github.com/jmix-framework/jmix/assets/6077149/0c6fac1e-0088-4573-96f0-3b6c5dd5024f">